dataprofiler/tests/labelers/test_integration_struct_data_labeler.py

```diff
@@ -267,6 +267,92 @@ def test_structured_data_labeler_fit_predict_take_data_obj(self):
+
+        results = profile.report()
+
+        # print(json.dumps(human_readable_report, indent=4))
+        columns = []
+        predictions = []
+        for col in results['data_stats']:
+            columns.append(col)
+            predictions.append(results['data_stats'][col]['data_label'])
+        print(columns)
+        print(predictions)
+
+
+    def test_warning_tf_run_dp_multiple_times(self):
+        test_root_path = os.path.dirname(os.path.dirname(os.path.realpath(__file__)))
+        test_dir = os.path.join(test_root_path, 'data')
+        path = os.path.join(test_dir, 'csv/diamonds.csv')
+
+        for i in range(3):
+            print('running dp =============================', i)
+            data = dp.Data(path)
+            profile_options = dp.ProfilerOptions()
+            profile_options.set({"text.is_enabled": False,
+                                 "int.is_enabled": False,
+                                 "float.is_enabled": False,
+                                 "order.is_enabled": False,
+                                 "category.is_enabled": False,
+                                 "datetime.is_enabled": False, })
+
+            profile = dp.Profiler(data, profiler_options=profile_options)
+
+            results = profile.report()
+
+            # print(json.dumps(human_readable_report, indent=4))
+            columns = []
+            predictions = []
+            for col in results['data_stats']:
+                columns.append(col)
+                predictions.append(results['data_stats'][col]['data_label'])
+            print(columns)
+            print(predictions)
+
+    def test_warning_tf_run_dp_merge(self):
+        test_root_path = os.path.dirname(os.path.dirname(os.path.realpath(__file__)))
+        test_dir = os.path.join(test_root_path, 'data')
+        path = os.path.join(test_dir, 'csv/diamonds.csv')
+
+        data = dp.Data(path)
+        profile_options = dp.ProfilerOptions()
+        profile_options.set({"text.is_enabled": False,
+                             "int.is_enabled": False,
+                             "float.is_enabled": False,
+                             "order.is_enabled": False,
+                             "category.is_enabled": False,
+                             "datetime.is_enabled": False, })
+        print('running dp1')
+        profile1 = dp.Profiler(data, profiler_options=profile_options)
+
+        data = dp.Data(path)
+        profile_options = dp.ProfilerOptions()
+        profile_options.set({"text.is_enabled": False,
+                             "int.is_enabled": False,
+                             "float.is_enabled": False,
+                             "order.is_enabled": False,
+                             "category.is_enabled": False,
+                             "datetime.is_enabled": False, })
+        print('running dp2')
+        profile2 = dp.Profiler(data, profiler_options=profile_options)
+
+        #profile = profile1 + profile2
+
```

Comment:
I think you need to also add a test where you do:

profile1 - create
profile2 - create
profile1 - update

To see if that triggers anything

Reply:
added